2015/03/06 11:06:23 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/html/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /var/www/html/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /var/www/html/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /var/www/html/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /var/www/html/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/html/yii/framework/gii/generators/model/ModelCode.php(60): CModule->__get('db')
#5 /var/www/html/yii/framework/web/CFormModel.php(39): ModelCode->init()
#6 /var/www/html/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/html/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 /var/www/html/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /var/www/html/dcageek/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/dcageek/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/dcageek/index.php?r=gii/default/index
---
2015/03/06 11:06:32 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/html/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /var/www/html/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /var/www/html/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /var/www/html/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /var/www/html/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/html/yii/framework/gii/generators/crud/CrudCode.php(44): CModule->__get('db')
#5 /var/www/html/yii/framework/web/CFormModel.php(39): CrudCode->init()
#6 /var/www/html/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/html/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#14 /var/www/html/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /var/www/html/dcageek/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/dcageek/index.php?r=gii/crud/index
HTTP_REFERER=http://localhost/dcageek/index.php?r=gii/default/index
---
2015/03/06 12:27:31 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/html/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /var/www/html/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /var/www/html/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /var/www/html/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /var/www/html/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/html/yii/framework/gii/generators/crud/CrudCode.php(44): CModule->__get('db')
#5 /var/www/html/yii/framework/web/CFormModel.php(39): CrudCode->init()
#6 /var/www/html/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/html/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#14 /var/www/html/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /var/www/html/dcageek/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/dcageek/index.php?r=gii/crud/index
HTTP_REFERER=http://localhost/dcageek/index.php?r=gii/controller/index
---
2015/03/06 12:29:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/html/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /var/www/html/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /var/www/html/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /var/www/html/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /var/www/html/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/html/yii/framework/gii/generators/crud/CrudCode.php(44): CModule->__get('db')
#5 /var/www/html/yii/framework/web/CFormModel.php(39): CrudCode->init()
#6 /var/www/html/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/html/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#14 /var/www/html/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /var/www/html/dcageek/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/dcageek/index.php?r=gii/crud/index
HTTP_REFERER=http://localhost/dcageek/index.php?r=gii/controller/index
---
2015/03/06 12:30:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/html/yii/framework/db/CDbConnection.php:399
Stack trace:
#0 /var/www/html/yii/framework/db/CDbConnection.php(347): CDbConnection->open()
#1 /var/www/html/yii/framework/db/CDbConnection.php(325): CDbConnection->setActive(true)
#2 /var/www/html/yii/framework/base/CModule.php(394): CDbConnection->init()
#3 /var/www/html/yii/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /var/www/html/yii/framework/gii/generators/crud/CrudCode.php(44): CModule->__get('db')
#5 /var/www/html/yii/framework/web/CFormModel.php(39): CrudCode->init()
#6 /var/www/html/yii/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /var/www/html/yii/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /var/www/html/yii/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /var/www/html/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /var/www/html/yii/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /var/www/html/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/html/yii/framework/web/CWebApplication.php(282): CController->run('index')
#13 /var/www/html/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#14 /var/www/html/yii/framework/base/CApplication.php(184): CWebApplication->processRequest()
#15 /var/www/html/dcageek/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/dcageek/index.php?r=gii/crud/index
HTTP_REFERER=http://localhost/dcageek/index.php?r=gii/controller/index
---
2015/03/07 00:22:45 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'jisda_system.user_login' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `user_login`.
in /var/www/html/dcageek/index.php (13)
2015/03/07 00:23:19 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'jisda_system.userlogin' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `userlogin`.
in /var/www/html/dcageek/index.php (13)
